.github/workflows/_rocm-test.yml

```diff
@@ -120,7 +134,7 @@ jobs:
           XLA_CLANG_CACHE_S3_BUCKET_NAME: ossci-compiler-clang-cache-circleci-xla
           PYTORCH_TEST_CUDA_MEM_LEAK_CHECK: ${{ matrix.mem_leak_check && '1' || '0' }}
           PYTORCH_TEST_RERUN_DISABLED_TESTS: ${{ matrix.rerun_disabled_tests && '1' || '0' }}
-        timeout-minutes: 270
+        timeout-minutes: ${{ fromJson(steps.test-timeout.outputs.timeout) }}
```

Comment:
I keep wondering what `fromJson` does here, converting string to number?

Reply:
yep, if its not there lint complains about timeout-minutes needing a number but the step output is a string